vmm/src/device_manager.rs

```diff
@@ -1219,9 +1397,11 @@ impl DeviceManager {
                     .map_err(DeviceManagerError::CreateVhostUserBlk)?,
             ));
 
-            self.add_migratable_device(
-                Arc::clone(&vhost_user_block_device) as Arc<Mutex<dyn Migratable>>
-            );
+            // Fill the device tree with a new node. In case of restore, we
+            // know there is nothing to do, so we can simply override the
+            // existing entry.
+            self.device_tree
```

Comment:
I'm not sure about all these duplicated comments. Not sure it adds anything to copy and paste everywhere.

Reply:
Ok I'll remove it.